Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -1164,6 +1164,9 @@ private void DisableControls(int mapIndex, int controlStartIndex, int numControl
                     SetInitialStateCheckPending(bindingStatePtr, false);
                 manager.RemoveStateChangeMonitor(controls[controlIndex], this, mapControlAndBindingIndex);
 
+                // Ensure that pressTime is reset if the composite binding is reenable. ISXB-505
+                bindingStatePtr->pressTime = default;
+
```

Comment:
I believe this is a good fix.
the `pressTime` field was central to the other bug I fixed, and so resetting it here makes perfect sense. I just wonder if there are _other_ places this fields should be reset.